docs/reference/remap/ipv6_to_ip.cue

```diff
@@ -0,0 +1,46 @@
+package metadata
+
+remap: functions: ipv6_to_ip: {
+	arguments: [
+		{
+			name:        "value"
+			description: "The IPv4 mapped IPv6 address to convert."
+			required:    true
+			type: ["string"]
+		},
+	]
+	return: ["string"]
+	category: "ip"
+	description: #"""
+		Converts an IPv4 mapped IPv6 address to an IPv4 address.
```

Comment:
Should this be named either `ipv6_to_ipv4` or shorter `ipv6_to_v4`? IPv6 addresses *are* IP addresses.